src/utils/process.ts

```diff
@@ -0,0 +1,12 @@
+import { exec } from 'child_process';
+
+export const execAsync = async (cmd: string): Promise<string> =>
+	new Promise((resolve, reject) => {
+		exec(cmd, (error, stdout) => {
+			if (error) {
+				reject(error);
+				return;
```

Comment:
You can directly return the reject.
```suggestion
				return reject(error);
```

Reply:
Applied.